examples/example-node-runner/package.json

```diff
@@ -10,11 +10,14 @@
     "lint": "biome check ./src",
     "lint:fix": "biome check --apply ./src",
     "typecheck": "tsc --noEmit",
-    "test": "node --test --loader ts-node/esm src/*.test.ts",
+    "test": "glob -c \"node --test --test-reporter=@voxpelli/node-test-pretty-reporter --loader ts-node/esm\" \"src/*.test.ts\"",
```

Comment:
The need for `glob` is unfortunate. This is because npm scripts on windows don't work well with globs, and `node:test` before node 22 doesn't support glob patterns natively.